merlin-framework/src/main/java/gov/nasa/jpl/aerie/merlin/framework/ThreadedTask.java

```diff
@@ -1,17 +1,26 @@
 package gov.nasa.jpl.aerie.merlin.framework;
 
+import gov.nasa.jpl.aerie.merlin.protocol.driver.CellId;
 import gov.nasa.jpl.aerie.merlin.protocol.driver.Scheduler;
+import gov.nasa.jpl.aerie.merlin.protocol.driver.Topic;
 import gov.nasa.jpl.aerie.merlin.protocol.model.Task;
 import gov.nasa.jpl.aerie.merlin.protocol.model.TaskFactory;
 import gov.nasa.jpl.aerie.merlin.protocol.types.Duration;
 import gov.nasa.jpl.aerie.merlin.protocol.types.TaskStatus;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Objects;
 import java.util.concurrent.ArrayBlockingQueue;
 import java.util.concurrent.Executor;
+import java.util.concurrent.ExecutorService;
+import java.util.function.Consumer;
 import java.util.function.Supplier;
 
 public final class ThreadedTask<Return> implements Task<Return> {
+  public static boolean CACHE_READS = Boolean.parseBoolean(getEnv("THREADED_TASK_CACHE_READS", "true"));
```

Comment:
Actually, this env var is useful to disable caching for regular merlin simulation.

Reply:
I thought Merlin-side simulation isn't currently caching, though? And if it was set up, it using an envvar like the `MAX_NB_CACHED_SIMULATION_ENGINE` would be more consistent.

Reply:
Merlin-side sim is not caching but `ThreadedTask` is used in both simulations so we need to set this parameter somehow. Maybe this can be taken care of by `SimulationFacade` ?

Reply:
> if cached runs are disabled, then the scheduler is unable to "fall back" to another implementation. Disabling caching in the scheduler falls to the MAX_NB_CACHED_SIMULATION_ENGINE envvar.

Recently, Adrien has implemented this "fall back" behavior - if `MAX_NB_CACHED_SIMULATION_ENGINES` is 1, then it pushes a single simulation engine forward in time without ever duplicating it, which we think should be identical to the old resumable simulation behavior.